### PR TITLE
Activity daily count test

### PR DESCRIPTION
**Pull request by aminlatifi**
Added test for DailyTransactionCountViewRepository repository getDailyCounts method

### CODE REVIEW

File: packages/backend/src/peripherals/database/activity/BlockTransactionCountRepository.ts [hunk 1]
```diff
@@ -43,7 +43,7 @@ export class BlockTransactionCountRepository extends BaseRepository {
 
   async deleteAll() {
     const knex = await this.knex()
-    return await knex('activity.block').delete()
+    return knex('activity.block').delete()
```
Comment (antooni) [inline packages/backend/src/peripherals/database/activity/BlockTransactionCountRepository.ts:46]: why is there a need for this change?

File: packages/backend/src/peripherals/database/activity/DailyTransactionCountViewRepository.test.ts [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+  DailyTransactionCountViewRepository,
+} from './DailyTransactionCountViewRepository'
+import { StarkexTransactionCountRepository } from './StarkexCountRepository'
+import { ZksyncTransactionRepository } from './ZksyncTransactionRepository'
+
+const PROJECT_A = ProjectId('project-a')
+const PROJECT_B = ProjectId('project-b')
+const PROJECT_C = ProjectId('project-c')
+
+describe(DailyTransactionCountViewRepository.name, () => {
```
Comment (antooni) [inline packages/backend/src/peripherals/database/activity/DailyTransactionCountViewRepository.test.ts:22]: nest the tests for `.getDailyCounts()` in another describe

File: packages/backend/src/peripherals/database/activity/DailyTransactionCountViewRepository.test.ts [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+
+    await repository.refresh()
+  })
+
+  it('view must be empty', async () => {
+    const result = await repository.getDailyCounts()
+    expect(result).toEqual([])
+  })
+
+  it('should return correct response for single project activities', async () => {
```
Comment (antooni) [inline packages/backend/src/peripherals/database/activity/DailyTransactionCountViewRepository.test.ts:55]:
```suggestion
  it('should return correct response for single project', async () => {
```

File: packages/backend/src/peripherals/database/activity/DailyTransactionCountViewRepository.test.ts [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+        [0, 1],
+        [1, 2],
+        [2, 7],
+      ].map(([day, count]) =>
+        getDailyTransactionCountRecord(PROJECT_A, day, count),
+      ),
+    )
+  })
+
+  it('should return correct response for multiple project activities', async () => {
```
Comment (antooni) [inline packages/backend/src/peripherals/database/activity/DailyTransactionCountViewRepository.test.ts:75]:
```suggestion
  it('should return correct response for multiple projects', async () => {
```

File: packages/backend/src/peripherals/database/activity/BlockTransactionCountRepository.ts (hunk 1 above)
Comment (aminlatifi) [reply packages/backend/src/peripherals/database/activity/BlockTransactionCountRepository.ts:46]: I tried to mimic the other repository tests ... That part was unnecessary so wanted to clean it

Comment (antooni) [reply packages/backend/src/peripherals/database/activity/BlockTransactionCountRepository.ts:46]: but in other repositories we have the `await`

Comment (aminlatifi) [reply packages/backend/src/peripherals/database/activity/BlockTransactionCountRepository.ts:46]: I updated all 🙃

**Review by antooni — APPROVED** [review]

**Comment by aminlatifi** [comment]
Resolve L2B-2790

**Pull request merged**